### DISCUSSION TITLE
Error using the example `gdal-hello-world` algorithm

### DISCUSSION THREAD

**Discussion by emileten** [MAAP-Project/Community#842] [Platform] [answered]
Submitted a DPS job in the ADE submitted UI : 

- algorithm `gdal_hello_world:main`
- allocation 32 GB
- output tif `test.tif`
- input tif `https://maap-ops-workspace.s3.amazonaws.com/shared/hopebear_ecu/data/Copernicus_DSM_COG_10_S90_00_W175_00_DEM.tif`
- outsize_percent `"30"` (quotes matter)

Two problems : 

- for `outsize_percent`, `30` does not work, I guess it's a type problem in the algorithm definition. With `"30"` (quotes) inputs are validated.
- after fixing this, DPS returns another error

```
Product failed to ingest to data store: Traceback (most recent call last):
  File "/home/ops/verdi/ops/hysds-1.1.10/hysds/dataset_ingest_bulk.py", line 826, in publish_datasets
    metrics, prod_json = ingest_to_object_store(
  File "/home/ops/verdi/ops/hysds-1.1.10/hysds/dataset_ingest_bulk.py", line 366, in ingest_to_object_store
    pub_path_url = r.getPublishPath()
  File "/home/ops/verdi/ops/hysds-1.1.10/hysds/recognize.py", line 173, in getPublishPath
    return self.recognized["publish"]["location"].format(
KeyError: 'tag'
```

Questions : 

- Is there a publicly shared algorithm config yaml for this algo that someone can point me to ? 
- Is there a public set of input parameters that work for this algorithm
- If not does it mean it's broken and if so, should we fix it or _not_ use it as an example ? 

It's not explicitly referenced in the docs as an example but it seems that people in the uwg are using it. 

cc @sujen1412 since I see that you published the above linked repository.

**Comment by sujen1412** [comment, marked as answer]
I think this algorithm requires a re-registration on my end. 
We also going to include this in the docs formally in the next update.

**Reply by emileten** [reply]
thanks I converted this to an issue
